azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellAdapter.kt

```diff
@@ -39,4 +46,8 @@ internal class BottomCellAdapter(context: Context) : RecyclerView.Adapter<Bottom
         }
         super.notifyDataSetChanged()
     }
+
+    override fun getItemViewType(position: Int): Int {
```

Comment:
can use kotlin shorthand/type inference on one line methods like this

override fun getItemViewType(position: Int) = bottomCellItems[position].itemType.ordinal

Reply:
Included in 27731b8